api/types/src/transaction.rs

```diff
@@ -301,6 +309,37 @@ impl
     }
 }
 
+impl
+From<(
+    &UserAutomatedTransaction,
+    TransactionInfo,
+    TransactionPayload,
+    Vec<Event>,
+    u64,
+)> for Transaction
+{
+    fn from(
+        (txn, info, payload, events, timestamp): (
+            &UserAutomatedTransaction,
+            TransactionInfo,
+            TransactionPayload,
+            Vec<Event>,
+            u64,
+        ),
+    ) -> Self {
+        // If at some point Aptos will utilize Automation transactions then here we will use as
```

Comment:
Since `Aptos` does not have automation as of now, perhaps we should use `Supra` here in this comment?

Reply:
In supra as mentioned we are not using Apots API types. I mentioned aptos here as I though at some point if this fork will be made public or merged with aptos main branch to have some info for them.